src/core/session.cpp

```diff
@@ -35,7 +32,7 @@ class standard_logger_impl : public logger_impl
         logStream_ = NULL;
     }
 
-    virtual void start_query(std::string const & query)
+    virtual void start_query(std::string const & query) override
```

Comment:
Would it be possible to avoid non-CMake-related changes in this PR please?

Reply:
Iirc this was necessary to resolve some compile issues. Same for other non-cmake changes.